xblock/django/request.py

```diff
@@ -1,9 +1,11 @@
 """Helpers for WebOb requests and responses."""
+from builtins import object
```

Comment:
I don't love that future monkeypatches builtins.  I'm tempted to use the more explicit `from future.builtins import object`.  This `object` lets you use python 3 dunder-methods in python 2 (like __next__, __str__ returning unicode, etc).